simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationStore.java

```diff
@@ -100,6 +101,14 @@ public void setStationCertificate(X509Certificate stationCertificate) {
         this.stationCertificate = stationCertificate;
     }
 
+    public List<X509Certificate> getStationCertificateChain() {
+        return Collections.unmodifiableList(stationCertificateChain);
+    }
+
+    public void setStationCertificateChain(List<X509Certificate> stationCertificateChain) {
```

Comment:
It breaks immutability. It's better to store immutable copy inside instead of returning it in getter